ci/infra/vmware/lb-instance.tf

```diff
@@ -160,3 +161,23 @@ resource "null_resource" "lb_wait_cloudinit" {
     ]
   }
 }
+
+resource "null_resource" "haproxy_ready" {
+  depends_on = ["null_resource.lb_wait_cloudinit"]
+  count      = "${var.lbs}"
+
+  provisioner "local-exec" {
+    environment = {
+      host = "${element(vsphere_virtual_machine.lb.*.guest_ip_addresses.0, count.index)}"
+    }
+
+    command = <<EOT
+apistatus=$(curl -s "$host:9000/stats;csv" | awk -F ',' '/apiserver,FRONTEND/{print$18}')
+while [ "$apistatus" != "OPEN" ]
```

Comment:
probably a good idea to add a max_retry value here

Reply:
Removed this since it isn't needed